<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSelector</class>
 <widget class="QWidget" name="ChartSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ChartSelector</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QGroupBox" name="songIdSelectorGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>songIdSelector.title</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fuzzySearchLineEdit">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>fuzzySearch.lineEdit.placeholder</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="packageComboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="songIdComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="songIdSelectorQuickActionsGroupBox">
        <property name="title">
         <string>songIdSelector.quickActions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="previousPackageButton">
           <property name="text">
            <string>songIdSelector.quickActions.previousPackageButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previousSongIdButton">
           <property name="text">
            <string>songIdSelector.quickActions.previousSongIdButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextSongIdButton">
           <property name="text">
            <string>songIdSelector.quickActions.nextSongIdButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextPackageButton">
           <property name="text">
            <string>songIdSelector.quickActions.nextPackageButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ratingClassGroupBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>ratingClassSelector.title</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="RatingClassSelector" name="ratingClassSelector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="resultsHorizontalLayout">
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>resetButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RatingClassSelector</class>
   <extends>QWidget</extends>
   <header>ui.implements.components.ratingClassSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
